### DISCUSSION TITLE
Webp glb file show break in ios browser.

### DISCUSSION THREAD

**Discussion by neciszhang** [donmccurdy/glTF-Transform#585] [Q&A] [answered]
Hi, I convert glb to webp file. Here is  [webp glb file](https://gw.alipayobjects.com/os/bmw-prod/88d34158-072c-4f28-9a88-e192fb328bae.glb)
![image](https://user-images.githubusercontent.com/16221107/169187937-d563d599-4067-4868-87a8-cf15b04c02b2.png)

**Comment by donmccurdy** [comment, marked as answer]
This model looks OK on my iPhone:

![IMG_3C476AAE74ED-1 2](https://user-images.githubusercontent.com/1848368/169200281-eb88801d-75a8-4280-82e9-05c18fa2b349.jpeg)


However, it's important to know that WebP textures are not supported in all browsers:

https://caniuse.com/webp

In particular, Safari in iOS < 14 and macOS < 11. For these browsers you'll need to use a model without WebP compression.
